### PR TITLE
feat: release 3.6.1

### PR DESCRIPTION
**Pull request by Revolyssup**
### Description

Fixes # (issue)

### Checklist

- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

**Comment by Revolyssup** [comment]
This commit to be added https://github.com/apache/apisix/pull/10312

File: CHANGELOG.md [hunk 1]
```diff
@@ -73,6 +74,23 @@ title: Changelog
 - [0.7.0](#070)
 - [0.6.0](#060)
 
+## 3.6.1
+
+### Change
+
+- change: creating core resources does not allow passing in `create_time` and `update_time`：[#10232](https://github.com/apache/apisix/pull/10232)
```
Comment (AlinsRan) [inline CHANGELOG.md:81]: Why include break change？

File: docs/zh/latest/CHANGELOG.md [hunk 2]
```diff
@@ -73,6 +74,23 @@ title: CHANGELOG
...
+### Core
+
+- :sunrise: feat: add token support for consul discovery：[#10278](https://github.com/apache/apisix/pull/10278)
+
+### Bugfix
+
+- fix: fix and optimize tls in upstream_schema：[#10269](https://github.com/apache/apisix/pull/10269)
+- fix: add name field in plugin_config schema for consistency：[#10315](https://github.com/apache/apisix/pull/10315)
+- fix: update schema description in cors.lua：[#10314](https://github.com/apache/apisix/pull/10314)
+- fix: use warn log for get_target_status failure：[#10156](https://github.com/apache/apisix/pull/10156)
```
Comment (AlinsRan) [inline docs/zh/latest/CHANGELOG.md:92]: zh?

File: CHANGELOG.md [hunk 3]
```diff
@@ -73,6 +74,23 @@ title: Changelog
...
+### Core
+
+- :sunrise: 壮举：将 lua-resty-healthcheck 升级到 3.2.0：[#10307](https://github.com/apache/apisix/pull/10307)
+
+### Bugfix
+
+- 修复：修复并优化upstream_schema中的tls：[#10269](https://github.com/apache/apisix/pull/10269)
+- 修复：在plugin_config架构中添加名称字段以保持一致性：[#10315](https://github.com/apache/apisix/pull/10315)
+- 修复：更新 cors.lua 中的模式描述：[#10314](https://github.com/apache/apisix/pull/10314)
+- 修复：使用警告日志获取 get_target_status 失败：[#10156](https://github.com/apache/apisix/pull/10156)
```
Comment (AlinsRan) [inline CHANGELOG.md:92]: en?

**Review by AlinsRan — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (Revolyssup) [reply CHANGELOG.md:92]: looks like there was a mix up. My bad. Fixing it

File: CHANGELOG.md (hunk 1 above)
Comment (Revolyssup) [reply CHANGELOG.md:81]: fixed it

File: CHANGELOG.md [hunk 4]
```diff
@@ -73,6 +74,16 @@ title: Changelog
 - [0.7.0](#070)
 - [0.6.0](#060)
 
+## 3.6.1
+
+### Bugfix
+
+- fix: fix and optimize tls in upstream_schema：[#10269](https://github.com/apache/apisix/pull/10269)
+- fix: add name field in plugin_config schema for consistency：[#10315](https://github.com/apache/apisix/pull/10315)
+- fix: update schema description in cors.lua：[#10314](https://github.com/apache/apisix/pull/10314)
```
Comment (kayx23) [inline CHANGELOG.md:83]:
```suggestion
- fix: update schema description for `cors` plugin：[#10314](https://github.com/apache/apisix/pull/10314)
```

File: docs/zh/latest/CHANGELOG.md [hunk 5]
```diff
@@ -73,6 +74,16 @@ title: CHANGELOG
 - [0.7.0](#070)
 - [0.6.0](#060)
 
+## 3.6.1
+
+### Bugfix
+
+- 修复：修复并优化 upstream_schema 中的 tls：[#10269](https://github.com/apache/apisix/pull/10269)
+- 修复：在 plugin_config 架构中添加名称字段以保持一致性：[#10315](https://github.com/apache/apisix/pull/10315)
+- 修复：更新 cors.lua 中的模式描述：[#10314](https://github.com/apache/apisix/pull/10314)
```
Comment (kayx23) [inline docs/zh/latest/CHANGELOG.md:83]:
```suggestion
- 修复：更新 `cors` 插件中的模式描述：[#10314](https://github.com/apache/apisix/pull/10314)
```

**Review by kayx23 — APPROVED** [review]

File: CHANGELOG.md [hunk 6]
```diff
@@ -73,6 +74,16 @@ title: Changelog
...
 - [0.6.0](#060)
 
+## 3.6.1
+
+### Bugfix
+
+- fix: fix and optimize tls in upstream_schema：[#10269](https://github.com/apache/apisix/pull/10269)
+- fix: add name field in plugin_config schema for consistency：[#10315](https://github.com/apache/apisix/pull/10315)
+- fix: update schema description for `cors` plugin：[#10314](https://github.com/apache/apisix/pull/10314)
+- fix: use warn log for get_target_status failure：[#10156](https://github.com/apache/apisix/pull/10156)
```
Comment (kayx23) [inline CHANGELOG.md:84]:
```suggestion
- fix: use warn logging level for get_target_status failure：[#10156](https://github.com/apache/apisix/pull/10156)
```

File: docs/zh/latest/CHANGELOG.md [hunk 7]
```diff
@@ -73,6 +74,16 @@ title: CHANGELOG
...
 - [0.6.0](#060)
 
+## 3.6.1
+
+### Bugfix
+
+- 修复：修复并优化 upstream_schema 中的 tls：[#10269](https://github.com/apache/apisix/pull/10269)
+- 修复：在 plugin_config 架构中添加名称字段以保持一致性：[#10315](https://github.com/apache/apisix/pull/10315)
+- 修复：更新 `cors` 插件中的模式描述：[#10314](https://github.com/apache/apisix/pull/10314)
+- 修复：使用警告日志获取 get_target_status 失败：[#10156](https://github.com/apache/apisix/pull/10156)
```
Comment (kayx23) [inline docs/zh/latest/CHANGELOG.md:84]:
```suggestion
- 修复：使用警告级别日志当获取 get_target_status 失败：[#10156](https://github.com/apache/apisix/pull/10156)
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -73,6 +74,16 @@ title: Changelog
 - [0.7.0](#070)
 - [0.6.0](#060)
 
+## 3.6.1
+
+### Bugfix
+
+- fix: fix and optimize tls in upstream_schema：[#10269](https://github.com/apache/apisix/pull/10269)
```
Comment (kayx23) [inline CHANGELOG.md:81]:
```suggestion
- fix: fix and optimize TLS in upstream_schema：[#10269](https://github.com/apache/apisix/pull/10269)
```

File: docs/zh/latest/CHANGELOG.md [hunk 9]
```diff
@@ -73,6 +74,16 @@ title: CHANGELOG
 - [0.7.0](#070)
 - [0.6.0](#060)
 
+## 3.6.1
+
+### Bugfix
+
+- 修复：修复并优化 upstream_schema 中的 tls：[#10269](https://github.com/apache/apisix/pull/10269)
```
Comment (kayx23) [inline docs/zh/latest/CHANGELOG.md:81]:
```suggestion
- 修复：修复并优化 upstream_schema 中的 TLS：[#10269](https://github.com/apache/apisix/pull/10269)
```

File: docs/zh/latest/CHANGELOG.md [hunk 10]
```diff
@@ -73,6 +74,16 @@ title: CHANGELOG
...
+- 修复：在 plugin_config 架构中添加名称字段以保持一致性：[#10315](https://github.com/apache/apisix/pull/10315)
```
Comment (kayx23) [inline docs/zh/latest/CHANGELOG.md:82]:
```suggestion
- 修复：在 `plugin_config` 架构中添加名称字段以保持一致性：[#10315](https://github.com/apache/apisix/pull/10315)
```

File: CHANGELOG.md [hunk 11]
```diff
@@ -73,6 +74,16 @@ title: Changelog
...
+- fix: add name field in plugin_config schema for consistency：[#10315](https://github.com/apache/apisix/pull/10315)
```
Comment (kayx23) [inline CHANGELOG.md:82]:
```suggestion
- fix: add name field in `plugin_config` schema for consistency：[#10315](https://github.com/apache/apisix/pull/10315)
```

**Review by kayx23 — APPROVED** [review]

**Comment by moonming** [comment]
please refer to https://github.com/apache/apisix/pull/10137

**Pull request closed without merging**